utils/functions

```diff
@@ -155,7 +155,7 @@ m0_default_xprt()
     echo $trans
 }
 
-## To get local nid address
+## Te get local nid address
```

Comment:
Why is this needed?

Reply:
Thanks Hua, It was a typo not an intended change. I have reverted it back.

Reply:
OK.